tldw_Server_API/app/services/claims_review_metrics_scheduler.py

```
@@ -96,59 +92,50 @@ async def run_claims_review_metrics_once(
+                        user_ids = []
+                for user_id in user_ids:
+                    try:
+                        processed += await asyncio.to_thread(
+                            agg_fn,
+                            db=media_db,
+                            target_user_id=str(user_id),
+                            report_date=report_date,
+                            lookback_days=lookback_val,
+                        )
+                    except _CLAIMS_REVIEW_METRICS_NONCRITICAL_EXCEPTIONS as exc:
+                        logger.warning(f"claims_review_metrics: aggregation failed for user {user_id}: {exc}")
```

Ruff (0.15.6)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)